### ISSUE TITLE
Kibana Experiment Error: "/tmp/outputs/Best_Parameter_Set/data: no such file or directory"

### ISSUE THREAD

**Issue by rahuja23** [kubeflow/kubeflow#6393]
Hello I am trying to run a Kubeflow pipeline in which I am using Kibana for Hyper-Parameter Tuning. But the experiment just get stuck at Kibana Step for 1hr and then fails. Following is the code for the Kibana step of the pipeline:
`def create_katib_experiment_task(experiment_name, experiment_namespace, Data):
    # Trial count specification.
    max_trial_count = 1
    max_failed_trial_count = 1
    parallel_trial_count = 1

    # Objective specification

    objective = V1beta1ObjectiveSpec(
        type="minimize",
        goal=0.001,
        objective_metric_name="loss"
    )
    # Algorithm specification.
    algorithm = V1beta1AlgorithmSpec(
        algorithm_name="random",
    )

    parameters = [
        V1beta1ParameterSpec(
            name="batch_size",
            parameter_type="int",
            feasible_space=V1beta1FeasibleSpace(
                min="8",
                max="32"
            ),
        )
    ]

    trial_spec= {
        "apiVersion": "kubeflow.org/v1",
        "kind": "PyTorchJob",
        "spec":{
            "pytorchReplicaSpecs": {
                "Chief": {
                    "replicas": 1,
                    "restartPolicy": "OnFailure",
                    "template": {
                        "metadata": {
                            "annotations": {
                                "sidecar.istio.io/inject": "false"
                            }
                        },
                        "spec": {
                            "containers": [
                                {
                                    "name": "trainer",
                                    "image":"racahu23/ml-blueprint_trainer:12",
                                    " imagePullPolicy": "Always",
                                    "command": ["python"],
                                    "arguments":["primary_trainer.py",
                                        "--input_dir=" + str(Data),
                                        "--num_labels=9",
                                        "--logging_dir=logs/",
                                        '--num_train_epochs=1',
                                        '--evaluation_strategy=epoch',
                                        '--per_device_train_batch_size= ${trialParameters.batchSize}',
                                        '--per_device_eval_batch_size=64',
                                        '--save_strategy=epoch',
                                        '--logging_strategy=epoch',
                                        '--eval_steps=100'],
                                }
                            ]
                        }
                    }
                },
                "Worker":{
                    "replicas": 1,
                    "restartPolicy": "OnFailure",
                    "template": {
                        "metadata": {
                            "annotations": {
                                "sidecar.istio.io/inject": "false"
                            }
                        }
                    },
                    "spec":{
                        "containers": [
                            {

                                "name": "trainer",
                                "image": "racahu23/ml-blueprint_trainer:12",
                                "imagePullPolicy": "Always",
                                "command": ["python"],
                                "arguments": ["primary_trainer.py",
                                              "--input_dir=" + str(Data),
                                              "--num_labels=9",
                                              "--logging_dir=logs/",
                                              '--num_train_epochs=1',
                                              '--evaluation_strategy=epoch',
                                              '--per_device_train_batch_size= ${trialParameters.batchSize}',
                                              '--per_device_eval_batch_size=64',
                                              '--save_strategy=epoch',
                                              '--logging_strategy=epoch',
                                              '--eval_steps=100'],
                            }
                        ]
                    }
                }
            }
        }
    }
    # Configure parameters for the Trial template.
    trial_template = V1beta1TrialTemplate(
        primary_container_name="trainer",
        trial_parameters=[
            V1beta1TrialParameterSpec(
                name="batchSize",
                description="Batch size for the model",
                reference="batch_size"
            ),
        ],
        trial_spec=trial_spec
    )
    # Create an Experiment from the above parameters.
    experiment_spec = V1beta1ExperimentSpec(
        max_trial_count=max_trial_count,
        max_failed_trial_count=max_failed_trial_count,
        parallel_trial_count=parallel_trial_count,
        objective=objective,
        algorithm=algorithm,
        parameters=parameters,
        trial_template=trial_template,

    )

    # Create the KFP task for the Katib Experiment.
    # Experiment Spec should be serialized to a valid Kubernetes object.
    katib_experiment_launcher_op = components.load_component_from_url(
        "https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kubeflow/katib-launcher/component.yaml")
    op = katib_experiment_launcher_op(
        experiment_name=experiment_name,
        experiment_namespace=experiment_namespace,
        experiment_spec=ApiClient().sanitize_for_serialization(experiment_spec),
        experiment_timeout_minutes=60,
        delete_finished_experiment=False
        )

    # Output container to print the results.
    op_out = dsl.ContainerOp(
        name="best-hp",
        image="library/bash:4.4.23",
        command=["sh", "-c"],
        arguments=["echo Best HyperParameters: %s" % op.output],
    )
    op_out.container.set_image_pull_policy("Always")
    op_out.container.set_termination_message_policy('FallbackToLogsOnError')
    op.container.set_image_pull_policy("Always")
    op.container.set_termination_message_policy('FallbackToLogsOnError')
    return op
`
and here is the screenshot of the error I am getting:
<img width="994" alt="Screen Shot 2022-03-04 at 14 58 20" src="https://user-images.githubusercontent.com/51020974/156776812-c7d5b92d-9334-4f3e-8c8c-db81b3a7329c.png">

**Comment by juliusvonkohout** [comment]
/close

There has been no activity for a long time. Please reopen if necessary.
